src/components/showvenues.js

```diff
@@ -18,7 +18,7 @@ export default class ShowVenues extends Component {
   render() {
     const venues = this.state.venues;
 
-    const list = venues.map((venue, index) => {
+    const list = venues && venues.map((venue, index) => {
```

Comment:
is this a one line if? havent seen one here before?

Reply:
Cheeky logical operator - I've been using them instead of if's in the render occasionally. I think I used this one so that it would return false rather than an error for the venues.map so I could see the flow. Could probably be refactored out I think